changelog.md

```diff
@@ -44,6 +44,7 @@
 * [1989](https://github.com/zeta-chain/node/pull/1989) - simplify `IsSendOutTxProcessed` method and add unit tests
 * [2013](https://github.com/zeta-chain/node/pull/2013) - rename `GasPriceVoter` message to `VoteGasPrice`
 * [2059](https://github.com/zeta-chain/node/pull/2059) - Remove unused params from all functions in zetanode
+* [2071](https://github.com/zeta-chain/node/pull/2071) - Modify chains struct to add all chain related information
```

Comment:
Can we add it in the top level in a new unrelease section? Since this will be part of v16
We can put `Version: v15.0.0` instead of the current `Unreleased` also, we should have done it earlier

Reply:
Yeah, I see what you mean, I think the v15 and v16 changes have got mixed up . Not everything under the the current unrelated section is part of v15 , Ill try to clean up the changelog file using the v15.0.0 tag

Reply:
4a82d4508b940968d94866afff563d07855dd33b

We would still need to add this to v16.0.0-rc , as per the release conversation today , 
Alternatively we can create a free v16.0.0-rc2 (or v16.0.0-rc1) once we emerge this pr in